examples/scenes/src/simple_text.rs

```diff
@@ -210,7 +217,7 @@ impl SimpleText {
 }
 
 fn to_font_ref(font: &Font) -> Option<FontRef<'_>> {
-    use vello::skrifa::raw::FileRef;
+    use skrifa::raw::FileRef;
```

Comment:
Is there much advantage to this import being here?

Reply:
I don't think so. That was where it already was, but I've moved it to the top.

Reply:
Ah, apologies. I probably didn't originally notice that this was only a change and not an addition.